Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist

```diff
@@ -466,6 +466,30 @@ A profile can consist of payloads with different version numbers. For example, c
 			<string>Verify Cloud Password</string>
 			<key>pfm_type</key>
 			<string>boolean</string>
+		</dict>
+				<dict>
+			<key>pfm_description</key>
+			<string>hostname of the page that has the password field.</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/twocanoes/xcreds/wiki/AdminGuide#idphostname</string>
+			<key>pfm_name</key>
+			<string>idpHostName</string>
+			<key>pfm_title</key>
+			<string>idpHostName</string>
```

Comment:
The point of the `pfm_title` is to be a more user friendly version of what this actually is / does.  So you can either add this user friendly title or remove this `pfm_title` key.

Reply:
I committed a change to fix this.